src/libsyntax/tokenstream.rs

```diff
@@ -532,12 +685,14 @@ impl TokenSlice {
     }
 
     /// Returns the RHS of an assigment.
-    pub fn maybe_assignment(&self) -> Option<&TokenSlice> {
+    pub fn maybe_assignment(&self) -> Option<TokenStream> {
```

Comment:
I think this function probably wants to be with attribute stuff rather than here.